example/index.ts

```diff
@@ -1,55 +1,37 @@
-import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
-
-interface UseGravityArgs {
-  moverMass: number;
-  attractorMass: number;
-  attractor: Vector<number>;
-}
-
-const applyForceForStep = (config: UseGravityArgs, mover: Entity) => {
-  const force = forceV({
-    ...config,
-    mover: mover.position
-  });
-
-  return applyForce({
-    force,
-    entity: {
-      mass: config.moverMass,
-      acceleration: mover.acceleration,
-      velocity: mover.velocity,
-      position: mover.position
+import { gravity, vector } from "../src";
+
+const el = document.createElement("div");
+document.body.appendChild(el);
+
+const positionAttractor = [500, 500] as vector<number>;
+const attractorEl = document.createElement("div");
+document.body.appendChild(attractorEl);
+
+el.style.height = "10px";
+el.style.width = "10px";
+el.style.borderRadius = "50%";
+el.style.background = "steelblue";
+
+attractorEl.style.height = "25px";
+attractorEl.style.width = "25px";
+attractorEl.style.borderRadius = "50%";
+attractorEl.style.background = "orange";
+attractorEl.style.left = `${positionAttractor[0]}px`;
+attractorEl.style.top = `${positionAttractor[1]}px`;
+attractorEl.style.position = "absolute";
+
+const [stop] = gravity({
```

Comment:
So here's the main idea with the API that `gravity` exposes. @littlemilkstudio and I also discussed the idea of returning a `controller` here as well, which would have methods for synchronously updating some internal animation state (i.e. `attractor` position). This would be useful for creating really interactive 2D animations that follow mouse move, or allow things like pausing and resuming animations arbitrarily, but that feels like a more advanced feature right now.